pages/api/globalSearch.ts

```diff
@@ -27,58 +28,73 @@ export const config = {
 export default async function handler(req: Request) {
   const { searchParams } = new URL(req.url)
   const q = searchParams.get('q')
```

Comment:
Can we use something a bit more descriptive other than q (like query for example)